### PR TITLE
[POAE-2605] specify yaml path when construct veloxToSubstraitPlanConvertor

### PR DESCRIPTION
**Pull request by chaojun-zhang**
### What changes were proposed in this pull request?

specify substrait yaml root path when construct veloxToSubstraitPlanConvertor

### Why are the changes needed?

bug fix

### Does this PR introduce _any_ user-facing change?

No
### How was this patch tested?

UT Passed
### Which label does this PR belong to?

veloxIntegration

### CODE REVIEW

File: cider-velox/src/substrait/VeloxPlanFragmentToSubstraitPlanOptions.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include "VeloxPlanFragmentToSubstraitPlanOptions.h"
+
+DEFINE_string(substrait_yaml_root_path, "", "Substrait extension root path");
```
Comment (winningsix) [inline cider-velox/src/substrait/VeloxPlanFragmentToSubstraitPlanOptions.cpp:24]: in which case, we should add this path? Let's add some example in project website. We can add it under [section](https://intel.github.io/BDTK/user.html).

Also let's combine this with other options. ```cider-velox/src/ciderTransformer/CiderPlanTransformerOptions.cpp```. How about calling it ```veloxCiderOption``` and put it under cider-velox/src/option path. We can do it in follow-up PR once https://github.com/intel/BDTK/pull/179/files merged.

File: cider-velox/src/substrait/VeloxPlanFragmentToSubstraitPlan.cpp [hunk 2]
```diff
@@ -175,4 +177,29 @@ bool VeloxPlanFragmentToSubstraitPlan::shouldAppendValuesNode(
          !std::dynamic_pointer_cast<const core::AbstractJoinNode>(sourceNode);
 }
 
+VeloxPlanFragmentToSubstraitPlan::VeloxPlanFragmentToSubstraitPlan() {
+  if (FLAGS_substrait_yaml_root_path != "") {
+    static const std::vector<std::string> extensionFiles = {
```
Comment (winningsix) [inline cider-velox/src/substrait/VeloxPlanFragmentToSubstraitPlan.cpp:182]: Do we need to match exactly the file names? Or just iterate over the files under ```FLAGS_substrait_yaml_root_path``` without touching files.

Comment (chaojun-zhang) [reply cider-velox/src/substrait/VeloxPlanFragmentToSubstraitPlan.cpp:182]: Once we merge W43 into main branch ,  these code will removed , currently just workaround

Comment (winningsix) [reply cider-velox/src/substrait/VeloxPlanFragmentToSubstraitPlan.cpp:182]: sounds good.

**Comment by winningsix** [comment]
Offline discussed, if WW43 merged, we will close this PR.

**Comment by winningsix** [comment]
@harborn @PenghuiJiao should we close this PR since WW43 has been rebased?

**Pull request closed without merging**